mayastor/src/rebuild/rebuild_impl.rs

```diff
@@ -438,6 +421,24 @@ impl RebuildJob {
 
         unsafe { &mut *global_instances.inner.get() }
     }
+
+    /// Open a bdev handle for the given uri
+    fn open_handle(
+        uri: &str,
+        read_write: bool,
+        claim: bool,
+    ) -> Result<BdevHandle, RebuildError> {
+        BdevHandle::open(
+            &bdev_get_name(uri).context(BdevInvalidURI {
+                uri: uri.to_string(),
+            })?,
+            read_write,
```

Comment:
might be nice to have enums here instead of bools, so the action of calling function is more obvious